core/loadpoint.go

```diff
@@ -1156,7 +1156,7 @@ func (lp *Loadpoint) pvMaxCurrent(mode api.ChargeMode, sitePower float64, batter
 
 	if mode == api.ModePV && lp.enabled && targetCurrent < minCurrent {
 		// kick off disable sequence
-		if sitePower >= lp.Disable.Threshold && (lp.phaseTimer.IsZero() || sitePower/float64(activePhases) >= lp.Disable.Threshold) {
+		if sitePower-minCurrent*float64(activePhases-1) >= lp.Disable.Threshold || (sitePower >= lp.Disable.Threshold && lp.phaseTimer.IsZero()) {
```

Comment:
Nitpick: Können wir hier bitte die Reihenfolge wieder zurückändern (original Logik zu erst), dann ist der Change kompakter und vermutlich auch besser zu verstehen.

Reply:
Keine gute Idee - die alte Berechnung war schlicht falsch.

Reply:
Ich glaube wir reden aneinander vorbei. Mir gehts nur rein um die Reihenfolge der boolschen Teilausdrücke, nicht um die Inhalte des neuen Teilausdrucks.

```suggestion
		if (sitePower >= lp.Disable.Threshold && lp.phaseTimer.IsZero()) || sitePower-minCurrent*float64(activePhases-1) >= lp.Disable.Threshold  {
```

Wie initial geschrieben, eine Kleinigkeit. Könnte aber helfen ggf. den Change irgendwann nochmal besser nachzuvollziehen. 🤷‍♂️

Reply:
Mittlerweile in Nebenrechnung, damit sollte die Logik klar sein.